### ISSUE TITLE
Effective license of `BSD-3-Clause AND BSD-3-Clause`

### ISSUE THREAD

**Issue by tsteenbe** [oss-review-toolkit/ort#8714]
ORT's LicenseResolver does not seem to be capable of resolving of SPDX expression where left and right side of an AND are the same 
![ort-license-choice-bug](https://github.com/oss-review-toolkit/ort/assets/435781/03e3d10d-b562-48f4-85e1-509ccd3f5822)

### To Reproduce

This [license-choice-bug.zip](https://github.com/oss-review-toolkit/ort/files/15475849/license-choice-bug.zip) contains all ORT results and reports for the test project I created from real world ORT scan to create a test case.

1. Set up [orthw](https://github.com/oss-review-toolkit/orthw) on your machine
2. Download this [evaluation-result.json](https://github.com/oss-review-toolkit/ort/files/15475819/evaluation-result.json)
3. Run `mkdir license-choice-bug && license-choice-bug`
4. Initialize orthw with the directory using `orthw init file://${PWD}/evaluation-result.json`
5. Generate WebApp report using `orthw report-webapp`

### Expected behavior

ORT results webapp report where the effective license for Maven:jakarta.activation:jakarta.activation-api:2.1.2 is show as just BSD-3-Clause

### Environment

* ORT version: 22.6.0-036.sha.56bc621
* Java version: JDK 11.0.22+7, running under Java 11.0.22.
* OS: Mac OS X

**Comment by sschuberth** [comment]
Agreed with @oss-review-toolkit/core-devs that we always want to simplify `AND` expressions like this; probably some "normalization" is required to post-process license choices.
